cli/src/tests/parser_test.rs

```diff
@@ -872,6 +872,48 @@ fn test_parsing_with_multiple_included_ranges() {
     );
 }
 
+#[test]
+fn test_parsing_with_included_range_at_the_beginning_of_file() {
+    let source_code = "<div>test</div>{_ignore_this_part_}";
+
+    let mut parser = Parser::new();
+    parser.set_language(get_language("html")).unwrap();
+
+    let end_byte = source_code.find("{_ignore_this_part_").unwrap();
+
+    let range_to_parse = Range {
+        start_byte: 0,
+        start_point: Point {
+            row: 10,
+            column: 12,
+        },
+        end_byte,
+        end_point: Point {
+            row: 10,
+            column: 12 + end_byte,
+        },
+    };
```

Comment:
Is it intentional that this `start_point` does not match the `start_byte`? The test doesn't appear to make assertions about the row/column positions of the nodes. If it's not necessary to have this mismatch, I think it'd be clearer if the points just matched the ranges (`start_point: Point { row: 0, column: 0 }, end_point: Point { row: 0, column: end_byte }`)

Reply:
It is intentional as I'm trying to provide a different start location.  This line asserts the column and lines match what was provided in set_included_ranges: `assert_eq!(html_tree.root_node().range(), range_to_parse,);`

Reply:
> @forbisc Thanks for the great bug report and PR. I left some minor comments.
> 
> Also, just for my understanding, if we _undid_ your fix, how exactly would your test fail?

It's this line here that would fail.  Maybe I could make that more clear by asserting the start_point instead of the whole range.  

assert_eq!(html_tree.root_node().range(), range_to_parse,);